integration_tests/test_dbapi.py

```diff
@@ -139,3 +139,11 @@ def test_session_properties(run_presto):
             assert value == '10m'
         elif prop == 'query_priority':
             assert value == '1'
+
+def test_transaction_single(presto_connection):
+    presto_connection.isolation_level = 1
```

Comment:
What would be its purpose? Would a property with a setter work if it is to provide a mutator?